<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowStudent</class>
 <widget class="QMainWindow" name="windowStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WindowStudent</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabStudentUpdatePasswd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1311</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabStudentProfile">
     <attribute name="title">
      <string>Profile</string>
     </attribute>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Student id:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Full name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of birth:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Country:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Registered date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_9">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Academic program:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStudentId">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentFullName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentGender">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentDateOfBirth">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentCountry">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentAddress">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentEmail">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentRegisteredDate">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentAcaProgram">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>150</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabStudentSchedule">
     <attribute name="title">
      <string>Academic Schedule</string>
     </attribute>
     <widget class="QGroupBox" name="gbStudentSchedule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>1281</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Schedule</string>
      </property>
      <widget class="QTableView" name="tvStudentSchedule">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>1261</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabStudentUpdatePassword">
     <attribute name="title">
      <string>Update Password</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxUpdatePass">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>531</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Update password</string>
      </property>
      <widget class="QLabel" name="labelRepeatNewPasswd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Repeat new password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNewPasswd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>New password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelOldPasswd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Old password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditOldPass">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNewPass">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNewPassRepeat">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelError">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelErrorOldPass">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnUpdatePasswd">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Update password</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
